lib/shipit.rb

```diff
@@ -50,6 +50,7 @@
 require 'shipit/github_http_cache_middleware'
 require 'shipit/cast_value'
 require 'shipit/line_buffer'
+require 'shipit/webhooks'
```

Comment:
I actually don't like that this is being moved out of the autoloading path, I see no reason for it. You say:

> The Webooks registry is out of the Rails autoloader path. In development it was quite often the case the the Rails reloader would re-initialize the extra_handlers attribute of the WebhooksController to an empty array - effectively un-registering all of the custom handlers. 

But that's because it should be registered from a `to_prepare` callback. Granted that's an easy mistake to make.

Reply:
Addressed by e03cefa